docs/source/user/api.rst

```diff
@@ -407,6 +407,67 @@ simple, but can be `enabled through configuration <./settings.html#openwisp-radi
 if identity verification is disabled for a particular org, an empty string
 will be acceptable.
 
+Registering to Multiple Organizations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An **HTTP 409** response will be returned if an existing user tries to register
+on a URL of different organization. The response will contain a list of organizations
+with which the user has already registered to the system. Following is an example
+response:
+
+.. code-block:: shell
+
+    $ curl -X POST \
+    http://127.0.0.1:8000/api/v1/radius/organization/openwisp/account/ \
+    -H 'content-type: application/json' \
+    -d '{
+        "username": "openwisp_user",
+        "email": "user@openwisp.io",
+        "password1": "<your-super-secret-password>",
+        "password2": "<your-super-secret-password>"
+    }'
+
+    HTTP/1.1 409 Conflict
+    Date: Tue, 21 Sep 2021 15:13:38 GMT
+    Server: WSGIServer/0.2 CPython/3.8.10
+    Content-Type: application/json
+    Vary: Accept, Origin
+    Allow: POST, OPTIONS
+    X-Frame-Options: DENY
+    Content-Length: 120
+    X-Content-Type-Options: nosniff
+    Referrer-Policy: same-origin
+
+    {"details":"A user like the one being registered already exists.","organizations":[{"slug":"default","name":"default"}]}
+
+The existing user can register with a new organization using the
```

Comment:
I'd clear this behavior could be disabled if needed, eg: `with a new organization (provided registration is enabled for that organization)`

Reply:
We haven't added a dedicated setting for this. Did you mean `provided registration is **disabled** for that organization)` ?

Reply:
I mean this (see the last field):

![Screenshot from 2021-09-22 10-03-13](https://user-images.githubusercontent.com/841044/134369270-43ef31dc-4a7f-4ce4-8d0f-8b7aba83a324.png)